Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-Microsoft_Defender_for_Endpoint_-_Malware_Detected.yml

```diff
@@ -0,0 +1,522 @@
+
+  Used Sub-playbooks:
+  * Enrichment for Verdict
+
+  If you wish to link this playbook to the relevant alerts automatically, we recommend using the following filters when configuring the playbook triggers: Alert Source = Correlation AND Alert Name = Malware detected by Microsoft Defender for Endpoint
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: dff7841b-d85a-4e59-893c-44b5388f18fe
+    type: start
+    task:
+      id: dff7841b-d85a-4e59-893c-44b5388f18fe
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "1"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 460,
+          "y": 170
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "1":
+    id: "1"
+    taskid: ee3f6422-3b2d-4f0e-8870-b0b0a0ce2787
+    type: title
+    task:
+      id: ee3f6422-3b2d-4f0e-8870-b0b0a0ce2787
+      version: -1
+      name: Enrich Indicators
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "2"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 460,
+          "y": 340
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "2":
+    id: "2"
+    taskid: 9d7b49be-2e04-4015-84dd-2ce8eee14869
+    type: playbook
+    task:
+      id: 9d7b49be-2e04-4015-84dd-2ce8eee14869
+      version: -1
+      name: Enrichment for Verdict
+      description: This playbook checks prior alert closing reasons and performs enrichment and prevalence checks on different IOC types. It then  returns the information needed to establish the alert's verdict.
+      playbookName: Enrichment for Verdict
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "3"
+    scriptarguments:
+      CloseReason:
+        simple: Resolved - False Positive,Resolved - Duplicate Incident,Resolved - Known Issue
+      Domain:
+        complex:
+          root: alert
+          accessor: domainname
+      FileSHA256:
+        complex:
+          root: alert
+          accessor: initiatorsha256
+      IP:
+        complex:
+          root: alert
+          accessor: hostip
+      URL:
+        complex:
+          root: alert
+          accessor: url
+      User:
+        complex:
+          root: alert
+          accessor: username
+      query:
+        simple: (initiatorsha256:${inputs.FileSHA256} or hostip:${inputs.IP}) and alertsource:${alert.sourceBrand} and alertname:${alert.name}
```

Comment:
if you are not using inputs, why you have it in the query?

Reply:
query parameters changed:
inputs.FileSHA256 -> alert.initiatorsha256
hostip:${alert.hostip}) -> hostname:${alert.hostname})